near-sdk/src/store/lookup_map/mod.rs

```diff
@@ -121,14 +117,15 @@ impl<K, V, H> fmt::Debug for LookupMap<K, V, H>
 where
     K: BorshSerialize + Ord,
     V: BorshSerialize,
-    H: CryptoHasher<Digest = [u8; 32]>,
+    H: StorageKeyer,
+    <H as StorageKeyer>::KeyType: AsRef<[u8]>,
 {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         f.debug_struct("LookupMap").field("prefix", &self.prefix).finish()
     }
 }
 
-impl<K, V> LookupMap<K, V, Sha256>
+impl<K, V> LookupMap<K, V, Identity>
```

Comment:
How would the user create a lookup map with a different keyer?

Reply:
with the `with_hasher` function (naming taken from HashMap, although API slightly different since they keep an initialized hasher in the map), but perhaps this should be named differently now that it also includes identity/unhashed bytes. wdyt?